### ISSUE TITLE
depth render

### ISSUE THREAD

**Issue by DailyInsistence** [AniMerPlus/AniMerPlus#4]
@torch.autocast(device_type='cuda', dtype=torch.float32)
    def __call__(self, vertices, faces):
        ''' Right now only render silhouettes and depths
            Input: 
            vertices: BN * V * 3
            faces: BN * F * 3
        '''
        # Split vertices to avoid out-of-memory
        silhouettes = []
        for vertice_, face_ in zip(vertices, faces):
            torch_mesh = Meshes(verts=vertice_.unsqueeze(0),
                                faces=face_.unsqueeze(0))
            silhouette = self.silhouette_renderer(meshes_world=torch_mesh.clone(),
                                                R=self.R, T=self.T)[...,3]
            silhouettes.append(silhouette)
        return torch.cat(silhouettes, dim=0)   How can I add code to this section to render depth information?
